### DISCUSSION TITLE
Add support for basic HTML tags

### DISCUSSION THREAD

**Discussion by playerofceleste** [DoubleDotStudios/Quartz#18] [Ideas]
eg \<p>, \<br>, \<h1> etc.

**Comment by vaelixd** [comment]
It's in the works. MDXEditor is good for basic markdown but not as good for Quartz's scenario. I'm working on a custom Markdown editor that should allow for full markdown support.

**Comment by vaelixd** [comment]
This was added in v1.2.0. This feature will stick around so I'm gonna close this discussion.
